libpod/container_config.go

```diff
@@ -381,10 +381,6 @@ type ContainerMiscConfig struct {
 	// restart the container. Used only if RestartPolicy is set to
 	// "on-failure".
 	RestartRetries uint `json:"restart_retries,omitempty"`
-	// PostConfigureNetNS needed when a user namespace is created by an OCI runtime
-	// if the network namespace is created before the user namespace it will be
-	// owned by the wrong user namespace.
-	PostConfigureNetNS bool `json:"postConfigureNetNS"`
```

Comment:
Is it worth leaving this in the struct and setting it unconditionally to true, to preserve compatibility in a downgrade case? Ensuring the bool is still in the DB and set to true guarantees containers created in 4.6 still behave the same if downgraded to 4.5

Reply:
How much do we care about that? I get the point and will add it back. The reason I am asking because I pay zero attention on this when reviewing PRs.

Reply:
We have not cared about downgrade in the past.